README.md

```diff
@@ -519,11 +493,24 @@ project setup with the code from the `KlaviyoSwiftExtension`. Below are instruct
 
 Once you have these three things, you can then use the push notifications tester and send a local push notification to make sure that everything was set up correctly.
 
+#### Badge Count
+>  ℹ️ Incrementing badge count support is available in SDK version [4.1.0](https://github.com/klaviyo/klaviyo-swift-sdk/releases/tag/4.1.0) and higher
```

Comment:
To ensure people don't think they can set badge counts in earlier versions, should we say "Setting or incrementing the badge count is available..."